src/main/resources/schemas/default.xml

```diff
@@ -589,7 +618,10 @@
         <option>both</option>
       </attribute>
     </tag>
-    <tag name="TITLEHEAD" where="primary">
```

Comment:
I'd say yes, since changing from one to the other requires some editorial thought. (to be clear, I *did* deprecate it--look down a few lines)